scripts/core/editor3/reducers/editor3.tsx

```diff
@@ -270,8 +317,30 @@ function escapeRegExp(string) {
  * @description Handle the editor tab key pressed event
  */
 const onTab = (state, e) => {
-    const {editorState} = state;
-    const newState = RichUtils.onTab(e, editorState, 4);
+    const {editorState, editorFormat = []} = state;
+    const selection = editorState.getSelection() as SelectionState;
+    const moreThanOneBlockSelected =
+        selection.getStartKey() !== selection.getEndKey();
+    const block = editorState
+        .getCurrentContent()
+        .getBlockForKey(selection.getStartKey()) as ContentBlock;
+    const blockType = block.getType();
+    let newState = editorState;
+
+    if (['unordered-list-item', 'ordered-list-item'].includes(blockType)) {
```

Comment:
can you explain what happens in both cases?

Reply:
Draft-js uses tab & shift+tab to indent/outdent an item on a list